orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/model/StageDefinition.java

```diff
@@ -41,6 +34,13 @@
   private List<String> when = new ArrayList<>();
   private InheritanceControl inheritanceControl;
   private Set<String> requisiteStageRefIds = new LinkedHashSet<>();
+  private String loopWith;
```

Comment:
Does this mean you can define one loop directive per stage? See comment in `loops-template.yml`